packages/softether-vpn/build.sh

```diff
@@ -0,0 +1,44 @@
+TERMUX_PKG_HOMEPAGE=https://www.softether.org/
+TERMUX_PKG_DESCRIPTION="An open-source cross-platform multi-protocol VPN program"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=(5.02.5180)
+TERMUX_PKG_VERSION+=(1.0.18)
+TERMUX_PKG_SRCURL=(https://github.com/SoftEtherVPN/SoftEtherVPN/releases/download/${TERMUX_PKG_VERSION}/SoftEtherVPN-${TERMUX_PKG_VERSION}.tar.xz
+                   https://github.com/jedisct1/libsodium/archive/${TERMUX_PKG_VERSION[1]}-RELEASE.tar.gz)
```

Comment:
Is libsodium sources used in compilation of softethervpn, or only for hostbuild step? Can't libsodium be [installed to the docker image](https://github.com/termux/termux-packages/blob/master/scripts/setup-ubuntu.sh) instead?

Reply:
Only for host build. Can be replaced by a binary package in docker.

Strictly, libsodium for host build should not be required at all. We only need to build the `hamcorebuilder` binary for running on build host, which does not depend on libsodium. But there seems to be no trivial way to build it separately.